docs/future/core/reference/corectl.md

```diff
@@ -228,6 +231,52 @@ Argument:
 
 * **address**: The listen address, in `host:port` format, to be added to the allowed members list.
 
+
+### `get`
+
+Retrieves the current value of a configuration option.
+
+```
+corectl get [key]
+```
+
+Argument:
+
+* **key**: The configuration option to retrieve.
+
+
+### `add`
+
+Adds a tuple to a configuration option's set of values. The configuration option must be defined as a set. The number of value arguments provided must match the configuration option's tuple length. Add will error if a conflicting tuple already exists in the set. To overwrite the existing tuple, provide the `-u` flag.
```

Comment:
I like minimal docs but I think this might not quite get the picture across, since our configuration scheme is a bit idiosyncratic.

YMMV, but here's a strawman. Instead of:

> Adds a tuple to a configuration option's set of values. The configuration option must be defined as a set. The number of value arguments provided must match the configuration option's tuple length.

we try something more explanatory, such as:

> Adds a tuple to a configuration option's set of values. 
> 
> A configuration option is identified by a key, and consists of a set of tuples. Tuples for a given configuration option always the contain the same number of elements. For example, each tuple in the `block_signers` configuration option must contain two elements, a URL and a public key.

And then in the docs for `rm` and `get`, we can link to `add` for an explanation:

```
See [`add`](#add) for a description of how configuration options are structured.
```